nixos/modules/system/boot/loader/grub/grub.nix

```diff
@@ -196,7 +196,7 @@ in
       };
 
       timeout = mkOption {
-        default = 5;
+        default = if (config.boot.loader.timeout != null) then config.boot.loader.timeout else -1;
```

Comment:
As said we can't deprecate all of them because it's not supported in
rename.nix. I'd say it's ok as is.

On Tue, Jan 20, 2015 at 12:10 AM, bobvanderlinden <notifications@github.com>
wrote:

> In nixos/modules/system/boot/loader/grub/grub.nix
> <https://github.com/NixOS/nixpkgs/pull/5772#discussion-diff-23193233>:
>
> > @@ -196,7 +196,7 @@ in
> >        };
> >
> >        timeout = mkOption {
> > -        default = 5;
> > +        default = if (config.boot.loader.timeout != null) then config.boot.loader.timeout else -1;
>
> This is also why I thought deprecating isn't a good idea. It's all still
> valid to use, but it might have non-obvious consequences. Is it better to
> just deprecate config.boot.loader.grub.timeout?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/5772/files#r23193233>.
>



-- 
NixOS Linux <http://nixos.org>